### ISSUE TITLE
Infinite Intellisense Loading...

### ISSUE THREAD

**Issue by ADSNB** [microsoft/TypeScript#57470]
Type: <b>Bug</b>

Basically when I open my nextjs with typescript project the intellisense works for the first 1~~5 seconds and after that the intellisense stop working. I can't navigate into objects, classes, functions, intellisense don't work anymore anywhere. The only thing that appears is "Loading..." when I hover any function/methods/objct/path and in the bottom corner is saying "Loading intellisense status...".

This issue is very annoying. Trying to develop a software without intellisense is just unproductive. In this past couple of days vscode has not been able to load the intellisense corretly at all in this project.

I've created this nextJS project using create-next-app with tailwindcss and typescript (following theire doc), I've added prisma, prisma class generators, nextUI to use in most of the UI components, trpc using querys with zod as validators, materialUI for icons, next-swagger-doc for swagger documentation.

I have seen a lot of topics over the internet about this issue and every single one of them has different ways to fix this and most of the responses of the people who got it working is "for now it's working" and we can't be sure it's have been fixed yet, because most of them don't reply the topic if the error is gone for good or they just moved on to another IDE.

I have tried a lot of things to try to fix this issue and still the intellisense is not working. Some of the things I tried are:

0 - Installed NodeJS Current then moved back to NodeJS LTS.
1 - Uninstall vs code and install a clean version.
2 - Install vs code insiders version.
3 - Start extension Bisect
4 - Clone entire repo again
5 - Force vscode to use typescript from the node_modules (project version) instead the workspace version.
6 - Clean cache folders located in "C:\Users\[USER]\AppData\Roaming\Code\Cache\Cache_Data" and "C:\Users\[USER]\AppData\Roaming\Code\CachedData".
7 - Restart TS server works for about 1~~5 seconds and then I lose intellisense functionality again.
8 - Trying to disable all extensions from VS code doesn't solve the issue.
9 - Enable TS server logs and check the logs to find issues. I have found some lines that could mean some issues, like:

"FileWatcher:: Added:: WatchInfo: c:/Projects/vega/tsconfig.json 2000 undefined"

"FileWatcher:: Added:: WatchInfo: c:/Projects/vega/tailwind.config.ts 500 undefined WatchType: Closed Script info"

"DirectoryWatcher:: Added:: WatchInfo: c:/Projects/vega/node_modules 1 undefined Project: c:/Projects/vega/tsconfig.json WatchType: Failed Lookup Locations"

But I'm not sure if this is normal messages or errors messages.

Any help is appreciated.

VS Code version: Code 1.86.1 (31c37ee8f63491495ac49e43b8544550fbae4533, 2024-02-07T09:08:20.941Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 3900XT 12-Core Processor            (24 x 3793)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.94GB (20.59GB free)|
|Process Argv|--crash-reporter-id f67be61d-5abc-48d3-82e2-d0a3448fac79|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.4.2
vscode-tailwindcss|bra|0.10.5
npm-intellisense|chr|1.4.5
path-intellisense|chr|2.8.5
vscode-eslint|dba|2.4.4
prettier-vscode|esb|10.1.0
code-runner|for|0.12.1
languague-renpy|Luq|2.2.2
git-graph|mhu|1.30.0
debugpy|ms-|2024.0.0
python|ms-|2024.0.1
vscode-pylance|ms-|2024.2.1
vscode-typescript-next|ms-|5.4.20240208
color-highlight|nau|2.5.0
indent-rainbow|ode|8.3.1
prisma|Pri|5.9.1
errorlens|use|3.16.0
vscode-icons|vsc|12.7.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscorecescf:30445987
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
vscaac:30438847
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
accentitlementsc:30887149
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
pydisww2:30959801
edj9j583:30958191
7j2b6412:30949929
ccp2r6:30958158

```

</details>

**Comment by RyanCavanaugh** [comment]
I've tried unsuccessfully to get a repro of this via `tsc` in latest, so this is probably something LS-specific

**Comment by JCMais** [comment]
@ADSNB were you able to find a workaround for this?

**Comment by ADSNB** [comment]
No, I haven`t found any workaround. I removed this library and stopped using, since it was consuming a lot of my time to try to fix this issue.

**Comment by jakebailey** [comment]
2 years on, TS7's LS is entirely redone and probably does not have this issue; feel free to recheck with TS7+ in VS Code.
